validator/src/commands/mod.rs

```diff
@@ -26,7 +28,8 @@ pub mod tests {
     {
         let matches = app.get_matches_from(vec);
         let result = T::from_clap_arg_match(&matches);
-        assert_eq!(result, expected_arg);
+        assert!(result.is_ok());
+        assert_eq!(result.unwrap(), expected_arg);
```

Comment:
nit: The `.is_ok()` check is redundant as `.unwrap()` will panic if `result` is an `Err(_)`

Reply:
🫡 https://github.com/anza-xyz/agave/pull/5011/commits/c1ba9a2dde510b2c55cf0a33824082fa308b3820